versions.gradle

```diff
@@ -183,7 +183,7 @@ deps.colorpicker = "com.github.skydoves:colorpickerpreference:2.0.6"
 ext.deps = deps
 
 def build_versions = [:]
-build_versions.min_sdk = 24
+build_versions.min_sdk = 34
```

Comment:
We can't directly upgrade the minimum SDK version to 34 since some VR systems still use old Android versions.

Instead, it would be best if you used a branch control for the new code, and kept using the old deprecated code when the SDK version goes below 34. e.g.:
https://github.com/Igalia/wolvic/blob/b02ebb2665e2a0d778bfa00579e7c613022f006e/app/src/oculusvr/java/com/igalia/wolvic/PlatformActivity.java#L90

Reply:
Thanks. I added a check for version >= 34